src/App.jsx

```diff
@@ -1,104 +1,23 @@
 import './App.scss';
-
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+const getUserId = userId =>
+  usersFromServer.find(user => user.id === userId || null);
+const getCommentsId = id =>
```

Comment:
```suggestion
const getCommentsByPostId = id =>
```
The same problem with naming